### PR TITLE
major: initial commit

### PR DESCRIPTION
**Pull request by Slesarew**

### CODE REVIEW

File: doc/yeast-uniformity.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+## Overview
+
+This test is performed on stored slants with yeast to increase certainty that population is indeed monoclonal and samplings from this storage unit are indeed genetically identical.
+
+## Prior knowledge
+
+Prior probability that culture is monoclonal: Pmp = 99%
+
+Mutation rate: km = 1e-7/s (P(mutation>1 | t) = 1 - exp(-t * km))
```
Comment (Zhenya-Barannik) [inline doc/yeast-uniformity.md:13]: I am not sure how to read line № 13.

File: data/liquid/2023/2023-05-30-2c.toml [hunk 2]
```diff
@@ -0,0 +1,55 @@
+id = "2023-05-30-2c"
+protocol.version = "0.2.0"
```
Comment (Zhenya-Barannik) [inline data/liquid/2023/2023-05-30-2c.toml:2]: Is the line
`protocol.name = "yeast-uniformity"`
missing?

File: doc/yeast-capture.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+On plating stage, we can assume zero-truncated (since we definitely observe a colony) Poisson distribution of number of strains in a colony, with probability of 1 colony at 80%. This results in zero-truncated Poisson parameters:
+
+P(1) = 0.8
+0.8 * (e^l - 1) = l
+l = 0.4
+
+This is approximate value giving 81% probability of 1-colony pickup; let's re-define it as actual prior, since underlying value was also given with less than 1 significant figure.
+
+Thus, after first pick-up, fraction of good cells in a tube is 1 with 81% probability, if it's 2 population, then fraction of good cells is uniformly distributed between 0.5 and 1 (since if it's less than 0.5, we name the other one "good"). And so forth.
+
```
Comment (Zhenya-Barannik) [inline doc/yeast-capture.md:77]: Let's use a common markdown style for links.

`[^White-Zainacheff]: Yeast, The Practical Guide to Beer Fermentation. ISBN-13: 978-0937381960.`

File: doc/yeast-capture.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+The colonies on plate are observed as circles, often isolated, indicating effective single-cell distribution.
+
+TODO: analyze plate photographs to obtain some information of colonies distribution
+
+TODO: estimate how well we could identify overlapping circles
+
+TODO: literature reference on this
+
+PRIOR
+Worst case estimate encountered in [Yeast: White, Zainacheff](ISBN-13: 978-0937381960) seems to be around 20% probability of pickup of colony from more than one origin cell. We will use worst-case for now.
```
Comment (Zhenya-Barannik) [inline doc/yeast-capture.md:34]: `Worst case estimate encountered in [^White-Zainacheff] seems to be around 20% probability of pickup of colony from more than one origin cell. `

**Comment by Zhenya-Barannik** [comment]
@Slesarew Didn't you plan to store raw data and rust code in different places?

File: doc/yeast-uniformity.md (hunk 1 above)
Comment (Slesarew) [reply doc/yeast-uniformity.md:13]: this is probability of mutations count being non-zero at time t

File: data/liquid/2023/2023-05-30-2c.toml (hunk 2 above)
Comment (Slesarew) [reply data/liquid/2023/2023-05-30-2c.toml:2]: yes, I've added it in today's dataset

**Comment by Zhenya-Barannik** [comment]
We decided to leave the code in this project for now.

**Review by Zhenya-Barannik — APPROVED** [review]
All is ok.

**Pull request merged**